.github/workflows/test.yaml

```diff
@@ -21,14 +21,11 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
+      - name: Install AWS CDK CLI
+        run: npm install -g aws-cdk
       - name: Install dependencies
         run: pip install -r requirements.txt -r requirements-dev.txt
       - name: Generate cloudformation
-        uses: youyo/aws-cdk-github-actions@v2
         env:
           ENV: ${{ inputs.environment }}
-        with:
-          cdk_subcommand: 'synth'
-          actions_comment: false
-          debug_log: true
-          cdk_args: '--output ./cdk.out'
+        run: cdk synth --output ./cdk.out
```

Comment:
It looks like we're losing debug output, how about adding `--debug true` to the cdk command?

https://docs.aws.amazon.com/cdk/v2/guide/ref-cli-cmd.html

Reply:
done